src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -74,6 +74,16 @@
   <strong>{{ actionDescription }}</strong> if you proceed.</ng-container>
 </ng-template>
 
+<ng-template #flagsTpl
+             let-value="value">
+  <div *ngFor="let flag of filterClusterWideFlags(clusterWideFlags);">
+    <span class="badge badge-hdd">{{ flag }}</span>
+  </div>
+  <div *ngFor="let flag of filterIndivFlags(value);">
```

Comment:
Following the previous comments:
This way angular is not recalculation the value of the functions on each check.
```suggestion
<ng-template #flagsTpl
             let-row="row">
  <div *ngFor="let flag of row.cdClusterFlags;">
    <span class="badge badge-hdd">{{ flag }}</span>
  </div>
  <div *ngFor="let flag of row.cdIndivFlags;">
```